core/jobs/batch_jobs/exp_migration_jobs.py

```diff
@@ -180,6 +185,48 @@ def _update_exp_summary(
 
         return updated_exp_summary_model
 
+    @staticmethod
+    def _extract_required_translation_model(
+        exp_translation_models: List[List[
+            translation_models.EntityTranslationsModel]],
```

Comment:
Why list of list?

Reply:
For the exploration having a translation model, it's a list_with_one_item(list(model)) and for cases, no translation model exists its empty list, so list[0] will raise indexError while calling the function.

I've added an inline if-else before calling the function to fix it!

Done!